keyboards/ploopyco/readme.md

```diff
@@ -33,13 +33,14 @@ The `DPI_CONFIG` macro will cycle through the values in the array, each time you
 
 ## Drag Scroll
 
-Drag Sroll is a custom keycode for the Ploopy devices that allow you to hold or tap a button and have the mouse movement translate into scrolling instead.
+The Drag Scroll custom keycodes for the Ploopy devices allow you to hold or tap a button and have the mouse movement translate into scrolling instead.
 
-Nothing needs to be enabled to use this functionality.  Just add the `DRAG_SCROLL` to your keymap.
+Nothing needs to be enabled to use this functionality.  Just add the `DRAG_SCROLL` or `TOGGLE_DRAG_SCROLL` to your keymap. `DRAG_SCROLL`
+defaults to being a momentary key, while `TOGGLE_DRAG_SCROLL` toggles scrolling.
 
 ### Drag Scroll Configuration
 
-* `#define PLOOPY_DRAGSCROLL_MOMENTARY` - Makes the key into a momentary key, rather than a toggle.
+* `#define PLOOPY_DRAGSCROLL_MOMENTARY` - Makes `DRAG_SCROLL` into a momentary key, rather than a toggle.
```

Comment:
I thought it best to preserve the behavior of this macro even though this is explicitly the use case for `TOGGLE_DRAG_SCROLL`. I wasn't sure what would be preferred, to preserve it or just delete the macro. Preserving it felt more conservative.